en/starter/generator/command_line_tools_for_linux.md

```diff
@@ -0,0 +1,235 @@
+| **`yo swiftserver:build`**        | [Build generator](#build-generator)              | Rebuild your Swift Server Generator application. |
+
+---
+
+## Application generator
+
+**Synopsis**
+
+Creates a new Swift Server Generator application.
+
+    yo swiftserver [options] [name]
+
+**Options**
+
+`-h, --help`
+
+Prints the generator’s options and usage.
+
+`--skip-build`
+
+Generates an application, but does not build it.
+
+`--single-shot`
+
+Disables creation of generator meta-data files (`.yo-rc.json` and `.swiftservergenerator-project`).
+Although, this means that you cannot run the generator again on your generated application.
+
+`--init`
+
+Generates a basic scaffolded application with default options into the current directory. The name
+parameter is ignored when using this option.
+
+**Interactive prompts**
+
+The generator prompts you for the following information:
+
+-   The name of the application to create, if you have not provided the application name as an argument, as in the following example:
+
+        yo swiftserver bookshop
+
+-   The name of the directory to create the application in; this defaults to the application name you previously entered.
+
+-   The type of project to create; this is a choice between scaffolding a starter project or generating a CRUD application.
+
+-   The pattern type which you would like your application to use; this defaults to Basic, you can also choose Web or Backend for Frontend (BFF).
+
+-   The capabilities which you would like to add to your application; these vary according to your project type. If you select a CRUD application type you can choose any/all or none of the Embedded metrics dashboard, Docker files or Bluemix cloud deployment capabilities. If you select scaffold a starter application you can choose additional capabilities of Static web file serving, OpenAPI/Swagger endpoint and Example endpoints.
+
+-   Select a Data store to use (for a CRUD application); this defaults to memory which is an in-memory data source that is built into Swift Server Generator and is suitable only for development and initial testing. If you want to connect your models to a real data source such as the Cloudant® database server, choose `cloudant` for your back-end data store. The tool prompts you for additional settings and automatically installs the appropriate Cloudant connector package by using the Swift Package Manager. If you want to switch between data stores see the [`config.json file`](config_json.html).
+
+-   Select services (for scaffold a starter application) to generate boilerplate for; these will be Bluemix services if you selected the Bluemix cloud deployment capability, otherwise they will be local services. The available Bluemix services are Cloudant®, Redis, Object Storage, AppID and Auto-scaling; the local services are CouchDB and Redis. The tool prompts you for additional settings and automatically installs the appropriate Swift packages using the Swift Package Manager.
```

Comment:
Remove reference to Bluemix capability. Change  "Bluemix services" to "IBM Cloud services" or "cloud services", whichever is more appropriate in each case.

Reply:
Done